### DISCUSSION TITLE
Limited geometry points are shown in spatial panel

### DISCUSSION THREAD

**Discussion by yuchuanjin** [dbeaver/dbeaver#21633] [Q&A]
Hi, when I try to visualize the geometry data in the spatial panel, it seems to limit the number of points on the map. I have 362806 points in the query results, but it doesn't show all these points on the map. The database I'm using is DuckDB. How could I show all the points on the map? Thanks for the help!

**Comment by LonwoLonwo** [comment]
Hello @yuchuanjin 

Maybe this setting will be helpful for you:

![2023-10-27 13_52_03-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/3c8e484a-d143-4e1b-b6ec-0c039a1b4fed)

**Reply by mkgrgis** [reply]
@yuchuanjin , 362806 points is unreal number for current web environment in DBeaver. Fast map is possible with not more than approximate 45000 points, see https://github.com/dbeaver/dbeaver/issues/12244
